examples/formatjson/formatjson.d

```diff
@@ -82,6 +84,8 @@ void main(string[] args)
         case EOF:
             putStr("***ERROR***");
             return;
+	case Symbol, Comment, Space:
+	    break;
```

Comment:
These should not ever happen, they should be an assert(0).

JSON5 tokens are not returned if you are not using JSON5 mode. Spaces could be included, but again, this code isn't selecting for them.

I have mixed feelings about including JSON5 tokens in the main enum, but it's so much cleaner than templatizing, and ugh, that gets messy.